aws-s3/scripts-ccao-data-warehouse-us-east-1/ccao-commercial_valuation.R

```diff
@@ -176,7 +172,9 @@ mutate(
     TRUE ~ yearbuilt
   ),
   tot_units = coalesce(tot_units, tot_apts, `boatslips`, `mobilehomepads`),
-  #marketvalue = coalesce(finalmarketvalue, marketvalue, marketvalue_incl_excessland),
+  # marketvalue = coalesce(
+  # finalmarketvalue, marketvalue, marketvalue_incl_excessland
+  # ),
```

Comment:
**[Nitpick, non-blocking]** This doesn't really matter, but in the spirit of linting: We can probably just delete these lines instead of commenting them out.